.github/workflows/lfric_test.yml

```diff
@@ -74,6 +74,16 @@ jobs:
         # than the latest release from pypi.
         # pip install external/fparser
         pip install .[test]
+        # Fetch the specified version of LFRic apps
+        source /apps/spack/psyclone-spack-25.3/spack-repo/share/spack/setup-env.sh
+        spack load fcm%gcc
+        fcm co -r ${LFRIC_APPS_REV} https://code.metoffice.gov.uk/svn/lfric_apps/main/trunk/ lfric_apps
+        # We need to patch lfric_core to resolve issues with the nvfortran compiler
```

Comment:
Does this comment belong at L86 really? Is there an associated LFRic ticket we should point to?

Reply:
No, everything from this line down can be removed when the issue if fixed, because fcm will fetch the lfric_core requested in dependencies.sh.

I added the LFRic ticket number